handler.go

```diff
@@ -15,8 +15,28 @@ func getHealth(c *gin.Context) {
 func getTasks(c *gin.Context) {
 }
 
-// TODO: Implement getTasksByID
+// getTaskByID godoc
```

Comment:
Is this the syntax that maps directly to Swagger?

Reply:
Yes it is